{
	"SessionID": "127.0.0.1:22",
	"Stdout": "docker: Error response from daemon: Conflict. The container name \"/mydb4\" is already in use by container \"75147533778ef2375129c0ecb1de436b9ec8d30bc50759e10cc2cb398d73b290\". You have to remove (or rename) that container to be able to reuse that name.\r\nSee 'docker run --help'.\r\n",
	"Error": ""
}